src/lib/tvgPictureImpl.h

```diff
@@ -205,6 +205,7 @@ struct Picture::Impl
             if (w) *w = max.x - min.x;
             if (h) *h = max.y - min.y;
         } else {
+            reload();
```

Comment:
Thinking about a better approach...

Two considerations by me.
1. Now we have no more async beneifts by recent reload() calls.
2. Unecessary reload for bitmap images...

Reply:
right, this has to be changed, asap

Reply:
@hermet I made some changes, but still, I don;t like this solution too much, it's not cleat in the code what is happening and why. but I need it, because if an svg file doesn;t have any viewbox and width/height information, we're taking these values from the bounding box of all its paints. so to get this information up to date, we need to ensure, that the loading process is completed. 

I wanted to handle this very specific case in the svg loader, but you didn;t agree back then - maybe we didn't understand each other - the [discussion](https://github.com/thorvg/thorvg/pull/1318#discussion_r1133008968).

please take a look at https://github.com/thorvg/thorvg/pull/1338 - this was +- the original proposition how to solve this problem.

Reply:
I htink it's much easier when this is handled internaly in the svg loader